---
title: "RNA-Seq-Based Transcriptomics Study to Investigate the Genes Governing Nitrogen Use Efficiency in Indian Wheat Cultivars"
collection: publications
permalink: /publication/2022-03-31-RNA-Seq-NUE,-differential-gene-expression
excerpt: 'High NUE (nitrogen use efficiency) has great practical significance for sustainable crop production. Wheat is one of the main cultivated crops worldwide for human food and nutrition. However, wheat grain productivity is dependent upon cultivars with high NUE in addition to the application of nitrogen fertilizers. In order to understand the molecular mechanisms exhibiting a high NUE response, a comparative transcriptomics study was carried out through RNA-seq analysis to investigate the gene expression that regulates NUE, in root and shoot tissue of N-efficient (PBW677) and N-inefficient (703) cultivars under optimum and nitrogen (N) stress. Differentially expressed gene analysis revealed a total of 2,406 differentially expressed genes (DEGs) present in both the contrasting cultivars under N stress. The efficient genotype PBW677 had considerably more abundant DEGs with 1,653 (903 roots +750 shoots) compared to inefficient cultivar PBW703 with 753 (96 roots +657 shoots). Gene ontology enrichment and pathway analysis of these DEGs suggested that the two cultivars differed in terms of adaptive mechanism. Gene enrichment analysis revealed that among the upregulated and downregulated genes the overrepresented and underrepresented gene categories belonged to biological processes like DNA binding, response to abiotic stimulus, photosynthesis, carbon fixation, carbohydrate metabolic process, nitrogen compound metabolic process, nitrate transport, and translation in cultivar PBW677, while the enriched biological processes were nucleosome assembly, chromatin remodeling, DNA packaging, lipid transport, sulfur compound metabolic process, protein modifications, and protein folding and refolding in N inefficient cultivar PBW703. We found several transcription factors (MYB, WRKY, RING finger protein, zinc finger protein, transporters, NRT1, amino acid transporters, sugar), protein kinases, and genes involved in N absorption, transportation, and assimilation to be highly expressed in high NUE cultivar PBW677. In our study, we report 13 potential candidate genes which showed alternate gene expression in the two contrasting cultivars under study. These genes could serve as potential targets for future breeding programs.'
date: 2022-03-31
venue: 'Frontier in Genetics'
clink: 'https://doi.org/10.3389/fgene.2022.853910'
purl: '/files/research/RNA-seq.pdf'
citation: 'Kaur, S., <b>Shamshad, M.</b>, Jindal, S., Kaur, A., Singh, S., sharma, A., &amp; Kaur, S. (2022). &quot;RNA-Seq-Based Transcriptomics Study to Investigate the Genes Governing Nitrogen Use Efficiency in Indian Wheat Cultivars&quot;. <i>Frontiers in Genetics</i>, 13(March), 1-15. Retrieved from https://doi.org/10.3389/fgene.2022.8539'
pubtype: 'journal'
---

<a href='/files/research/RNA-seq.pdf'>Download PDF here</a>

Abstract: High NUE (nitrogen use efficiency) has great practical significance for sustainable crop production. Wheat is one of the main cultivated crops worldwide for human food and nutrition. However, wheat grain productivity is dependent upon cultivars with high NUE in addition to the application of nitrogen fertilizers. In order to understand the molecular mechanisms exhibiting a high NUE response, a comparative transcriptomics study was carried out through RNA-seq analysis to investigate the gene expression that regulates NUE, in root and shoot tissue of N-efficient (PBW677) and N-inefficient (703) cultivars under optimum and nitrogen (N) stress. Differentially expressed gene analysis revealed a total of 2,406 differentially expressed genes (DEGs) present in both the contrasting cultivars under N stress. The efficient genotype PBW677 had considerably more abundant DEGs with 1,653 (903 roots +750 shoots) compared to inefficient cultivar PBW703 with 753 (96 roots +657 shoots). Gene ontology enrichment and pathway analysis of these DEGs suggested that the two cultivars differed in terms of adaptive mechanism. Gene enrichment analysis revealed that among the upregulated and downregulated genes the overrepresented and underrepresented gene categories belonged to biological processes like DNA binding, response to abiotic stimulus, photosynthesis, carbon fixation, carbohydrate metabolic process, nitrogen compound metabolic process, nitrate transport, and translation in cultivar PBW677, while the enriched biological processes were nucleosome assembly, chromatin remodeling, DNA packaging, lipid transport, sulfur compound metabolic process, protein modifications, and protein folding and refolding in N inefficient cultivar PBW703. We found several transcription factors (MYB, WRKY, RING finger protein, zinc finger protein, transporters, NRT1, amino acid transporters, sugar), protein kinases, and genes involved in N absorption, transportation, and assimilation to be highly expressed in high NUE cultivar PBW677. In our study, we report 13 potential candidate genes which showed alternate gene expression in the two contrasting cultivars under study. These genes could serve as potential targets for future breeding programs.

Recommended citation: Kaur, S.[?term1], <b>Shamshad, M.</b>[?term1], Jindal, S., Kaur, A., Singh, S., sharma, A., & Kaur, S. (2022). "RNA-Seq-Based Transcriptomics Study to Investigate the Genes Governing Nitrogen Use Efficiency in Indian Wheat Cultivars". <i>Frontiers in Genetics</i>, 13(March), 1-15. Retrieved from https://doi.org/10.3389/fgene.2022.8539

[?term1]: Equal authorship